### ISSUE TITLE
[Feature Request]  Implement Bonjour/DNS-SD for easy service discovery

### ISSUE THREAD

**Issue by acutbal** [arpitkh96/AmazeFileManager#561]
Hi!!
I think that a really interesting feature to add to Amaze File Manager is the easy discovery when the ftp server is on. It's really useful, you only have to start the toggle and your phone will appear under Network tab in your PC's file managers. There are two apps which already have this option:

FTP Server :
https://f-droid.org/repository/browse/?fdfilter=ftp&fdid=be.ppareit.swiftp_free
https://play.google.com/store/apps/details?id=be.ppareit.swiftp

Primitive FTPd
https://f-droid.org/repository/browse/?fdfilter=ftp&fdid=org.primftpd
https://play.google.com/store/apps/details?id=org.primftpd

Both are open source, so I think that it couldn't be too much difficult to port it to Amaze. Also, it could be offered as a “premium feature” to do more appealing the paid plugin.

Regards!! :smile:

**Comment by EmmanuelMess** [comment]
@VishalNehra we have a discovery service for SMB servers, but I never saw it work. Is this considered fixed?

**Comment by VishalNehra** [comment]
Nope, it does return a series of stacktrace at failing something. We should fix this.

**Comment by EmmanuelMess** [comment]
> Nope, it does return a series of stacktrace at failing something. Although this issue isn't regarding that I think.

That would be issue #488 ?
